### PR TITLE
Update Codesniffer to version 2.* and Add CodeSniffer Code Beautifier and Fixer script

### PR DESCRIPTION
**Pull request by Krasnyj**

### CODE REVIEW

File: core/cibox-project-builder/files/drupal7/scripts/sniffers.yml [hunk 1]
```diff
@@ -30,11 +30,12 @@
     build_reports_dir: build_reports
 
     composer_global_require:
-      - drupal/coder:dev-7.x-2.x
-      - squizlabs/php_codesniffer=1.5.6
+      - { package: 'drupal/coder', version: ':dev-8.x-2.x' }
```
Comment (podarok) [inline core/cibox-project-builder/files/drupal7/scripts/sniffers.yml]: We can't set coder 8 for Drupal 7 scripts @Krasnyj 
Please use drupal8 folder which is already having Drupal 8 support

File: core/cibox-project-builder/files/drupal7/scripts/sniffers.yml [hunk 2]
```diff
@@ -30,11 +30,12 @@
...
+      - { package: 'squizlabs/php_codesniffer', version: '=2.*' }
+
     git_repos:
       - { branch: 'master', repo: 'http://git.drupal.org/sandbox/coltrane/1921926.git', name: 'DrupalSecure' } # git clone --branch master http://git.drupal.org/sandbox/coltrane/1921926.git DrupalSecure
-      - { branch: '7.x-1.x', repo: 'https://github.com/klausi/drupalpractice.git', name: 'DrupalPractice' } #git clone --branch 7.x-1.x https://github.com/klausi/drupalpractice.git DrupalPractice
+      - { branch: '8.x-1.x', repo: 'https://github.com/klausi/drupalpractice.git', name: 'DrupalPractice' } #git clone --branch 8.x-1.x https://github.com/klausi/drupalpractice.git DrupalPractice
```
Comment (podarok) [inline core/cibox-project-builder/files/drupal7/scripts/sniffers.yml]: We can't set coder 8 for Drupal 7 scripts @Krasnyj 
Please use drupal8 folder which is already having Drupal 8 support

**Comment by Sanchiz** [comment]
ok to test

File: core/cibox-project-builder/files/drupal7/scripts/sniffers.yml (hunk 2 above)
Comment (ygerasimov) [inline core/cibox-project-builder/files/drupal7/scripts/sniffers.yml]: @podarok 8.x branch can be used for testing D7 code. Please check https://www.drupal.org/node/2387871

Speicifcally https://monosnap.com/file/osyRqzyd8UcX13o7pGqlgzjxggyeSW

**Comment by podarok** [comment]
@Krasnyj and @ygerasimov 
Ok, agree with switching to 8.x
But we should upgrade core as well https://github.com/cibox/cibox/blob/master/core/cibox-sniffers/defaults/main.yml

**Comment by Krasnyj** [comment]
@podarok it is updated within my changes. The third file in my PR

**Comment by podarok** [comment]
Ah, sorry, missed that part.

**Comment by podarok** [comment]
Thanks @Krasnyj @ygerasimov

**Pull request merged**